core/assets/bundles/bundle.properties

```diff
@@ -649,6 +649,7 @@ stat.minetier = Mine Tier
 stat.payloadcapacity = Payload Capacity
 stat.commandlimit = Command Limit
 stat.abilities = Abilities
+stat.none =
```

Comment:
Put an empty key to each bundle is pretty strange.

Reply:
it's valid for .propeties

Reply:
Why do you even need an *empty* bundle key, bundles are for translations.

Reply:
becuz Stat.bla matches with bundle
if there is no bundle for it, it will show ???stat.none???

Reply:
or you can hardcode it into the display to make it just display nothing

Reply:
hmm.. ok